api/server/sdk/volume_ops.go

```diff
@@ -155,6 +155,16 @@ func (s *VolumeServer) create(
 				err.Error())
 		}
 
+		if spec.IsPureVolume() {
```

Comment:
was it not possible to do this at the filter or porx driver layer? We typically try to avoid driver-specific things in the SDK layer

Reply:
I'll think about that a little more, I think we went through the code together and thats how we concluded, may need to revisit.

Reply:
Sorry yeah I think we agreed that too. Lets try in the filter layer first as it's more ideal. We try to handle things like this in the driver layer.

Reply:
so I think the reason I added this check is specifically for restoring. for restoring, we are creating a new PVC from a snapshot, without this check, we go directly to create a new snapshot. I could add a check in creating snapshot and direct that to restore, but I dont know if that makes more sense. what are your thoughts?